packages/core/src/common/classes.ts

```diff
@@ -447,7 +447,7 @@ export function positionClass(position: Position | undefined) {
 }
 
 export function sizeClass(
-    size: Size | Omit<Size, "small">,
+    size: Size | NonSmallSize,
```

Comment:
This is equivalent to just `Size`, we can simplify this.

Reply:
0ebbacc70a325ddf7ec1a694ad492a89351698cb